specs/beacon-chain.md

```diff
@@ -746,7 +746,7 @@ For each `SpecialRecord` `obj` in `active_state.pending_specials`:
 
 * **[covers logouts]**: If `obj.kind == LOGOUT`, interpret `data[0]` as a validator index as an `uint32` and `data[1]` as a signature. If `BLSVerify(pubkey=validators[data[0]].pubkey, msg=hash(LOGOUT_MESSAGE + bytes8(fork_version)), sig=data[1])`, where `fork_version = pre_fork_version if slot < fork_slot_number else post_fork_version`, and `validators[i].status == ACTIVE`, run `exit_validator(data[0], crystallized_state, penalize=False, current_slot=block.slot)`
```

Comment:
(probably off this PR)
1. The `data` in this section looks like `obj.data` to me, and I *think* we either (i) wrongly used `data[order]` to describe the sequence of the fields in `data` or (ii) we should use another expression for data deserialization.
2. If it's (ii): since `uint32` is 4 bytes, should it be:
interpret `data[0:4]` as a validator index as an `uint32` and `data[4:]` as a signature.